app/workers/company_worker.rb

```diff
@@ -0,0 +1,24 @@
+class CompanyWorker
+  include Sidekiq::Worker
+
+  def perform(company, count)
+    result = GlassdoorService.new(GlassdoorParser).company(company)
+    if result
+      Company.find_by(name: company).update_attributes(
+        url: result["website"],
+        industry: result["industry"],
+        logo_url: result["squareLogo"],
+        overall_rating: result["overallRating"].to_f,
+        rating_description: result["ratingDescription"],
+        cultureandvalues_rating: result["cultureAndValuesRating"].to_f,
+        seniorleadership_rating: result["seniorLeadershipRating"].to_f,
+        compensationandbenefits_rating: result["compensationAndBenefitsRating"].to_f,
```

Comment:
If anyone has great ideas about how to make this cleaner, I'd love to hear about it.